docs/content/node-operation/node-setup.mdx

```diff
@@ -131,16 +131,208 @@ Replace `$ROLE` with the node type you are planning to run. Valid options are:
+
+### Docker Configuration
+
+If you are not using Systemd, sample commands for running each Docker container are below.
+Be sure to replace `/path/to/data` and `/path/to/bootstrap` with the appropriate paths you are using.
+
+<Callout type="warning">
+Do not run your node using `docker run` command directly without a mechanism for the node
+to automatically restart following a crash.
+</Callout>
+
+### Access
+
+```shell
+docker run --rm \
+	-v /path/to/bootstrap:/bootstrap:ro \
+	-v /path/to/data:/data:rw \
+	--name flow-go \
+	--network host \
+	gcr.io/flow-container-registry/access:v0.24.4 \
+	--nodeid=${FLOW_GO_NODE_ID} \
+	--bootstrapdir=/bootstrap \
+	--datadir=/data/protocol \
+	--secretsdir=/data/secrets \
+	--rpc-addr=0.0.0.0:9000 \
+	--http-addr=0.0.0.0:8000 \
+	--collection-ingress-port=9000 \
+	--script-addr=${FLOW_NETWORK_EXECUTION_NODE} \
+	--bind 0.0.0.0:3569 \
+	--loglevel=error
+```
+
+### Collection
+
+```shell
+docker run --rm \
+	-v /path/to/bootstrap:/bootstrap:ro \
+	-v /path/to/data:/data:rw \
+	--name flow-go \
+	--network host \
+	gcr.io/flow-container-registry/collection:v0.24.4 \
+	--nodeid=${FLOW_GO_NODE_ID} \
+	--bootstrapdir=/bootstrap \
+	--datadir=/data/protocol \
+	--secretsdir=/data/secrets \
+	--ingress-addr=0.0.0.0:9000 \
+	--bind 0.0.0.0:3569 \
+	--loglevel=error
+```
+
+### Consensus
+
+```shell
+docker run --rm \
+	-v /path/to/bootstrap:/bootstrap:ro \
+	-v /path/to/data:/data:rw \
+	--name flow-go \
+	--network host \
+	gcr.io/flow-container-registry/consensus:v0.24.4 \
+	--nodeid=${FLOW_GO_NODE_ID} \
+	--bootstrapdir=/bootstrap \
+	--datadir=/data/protocol \
+	--secretsdir=/data/secrets \
+	--bind 0.0.0.0:3569 \
+	--loglevel=error
+```
+
+### Execution
+
+```shell
+docker run --rm \
+	-v /path/to/bootstrap:/bootstrap:ro \
+	-v /path/to/data:/data:rw \
+	--name flow-go \
+	--network host \
+	gcr.io/flow-container-registry/execution:v0.24.4 \
+	--nodeid=${FLOW_GO_NODE_ID} \
+	--bootstrapdir=/bootstrap \
+	--datadir=/data/protocol \
+	--secretsdir=/data/secrets \
+	--triedir=/data/execution \
+	--rpc-addr=0.0.0.0:9000 \
+	--bind 0.0.0.0:3569 \
+	--loglevel=error
+```
+
+### Verification
 
 ```shell
- mv ~/flow-${ROLE}.service /etc/systemd/system/.\n\nsystemctl enable flow
+docker run --rm \
+	-v /path/to/bootstrap:/bootstrap:ro \
+	-v /path/to/data:/data:rw \
+	--name flow-go \
+	--network host \
+	gcr.io/flow-container-registry/verification:v0.24.4 \
+	--nodeid=${FLOW_GO_NODE_ID} \
+	--bootstrapdir=/bootstrap \
+	--datadir=/data/protocol \
+	--secretsdir=/data/secrets \
+	--bind 0.0.0.0:3569 \
+	--loglevel=error
 ```
 
-If you are using some other system, you need to ensure that the Flow container is started, and the appropriate key directories are mounted into the container.
+## Start the Node
+
+Now that your node is provisioned and configured, it can be started.
+
+<Callout type="warning">
+Before starting your node, ensure it is registered and authorized.
```

Comment:
Might be good to include a link here to those instructions

Reply:
👍  Added in https://github.com/onflow/flow/pull/862/commits/d1882fd04d82ed48f2d1085039ef9e8868f70a39